src/main/java/seedu/address/logic/parser/FindCustomerCommandParser.java

```diff
@@ -2,14 +2,12 @@
 
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 
-import java.util.Arrays;
-
 import seedu.address.logic.commands.FindCustomerCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.CustomerNameContainsStringPredicate;
 
 /**
- * Parses input arguments and creates a new FindCommand object
+ * Parses input arguments and creates a new FindTaskCommand object
  */
 public class FindCustomerCommandParser implements Parser<FindCustomerCommand> {
 
```

Comment:
for javadocs, should be FindCustomerCommand instead of FindTaskCommand

pls look through even when copying